conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -50,6 +50,7 @@ dependencies:
   - hypothesis
   - sphinx-markdown-tables
   - sphinx-copybutton
+  - sphinxemoji
```

Comment:
Need to add this to the `rapids-doc-env` as well: https://github.com/rapidsai/integration/blob/branch-0.15/conda/recipes/rapids-doc-env/meta.yaml

Reply:
Added to integration repo : https://github.com/rapidsai/integration/pull/90